general/device.cpp

```diff
@@ -33,14 +33,15 @@ occa::device occaDevice;
 static const Backend::Id backend_list[Backend::NUM_BACKENDS] =
 {
    Backend::OCCA_CUDA, Backend::RAJA_CUDA, Backend::CUDA,
+   Backend::HIP,
    Backend::OCCA_OMP, Backend::RAJA_OMP, Backend::OMP,
    Backend::OCCA_CPU, Backend::RAJA_CPU, Backend::CPU
 };
 
 // Backend names listed by priority, high to low:
 static const char *backend_name[Backend::NUM_BACKENDS] =
 {
-   "occa-cuda", "raja-cuda", "cuda", "occa-omp", "raja-omp", "omp",
+   "occa-cuda", "raja-cuda", "cuda", "hip", "occa-omp", "raja-omp", "omp",
    "occa-cpu", "raja-cpu", "cpu"
```

Comment:
Update also the documentation of `Device:: Configure`?

Reply:
Done